docs/sample-out/section=123456789/grades/2020-09-18-15-04-24.csv

```diff
@@ -1,5 +1,5 @@
 ﻿SourceSystemIdentifier,SourceSystem,Grade,GradeType,LMSUserLMSSectionAssociationSourceSystemIdentifier,CreateDate,LastModifiedDate,SourceCreateDate,SourceLastModifiedDate
-11357982,Canvas,A,Current,123456,,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-11357983,Canvas,90,Current,123457,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-11357984,Canvas,69,Final,123458,2020-08-20 12:34:56,2020-08-20 12:34:56,,
-11357985,Canvas,C,Final,123459,2020-08-20 12:34:56,2020-08-20 12:34:56,,
+11357982,Canvas,A,Current,123456,8/20/2020 12:34,8/20/2020 12:34,,
+11357983,Canvas,90,Current,123457,8/20/2020 12:34,8/20/2020 12:34,,
+11357984,Canvas,69,Final,123458,8/20/2020 12:34,8/20/2020 12:34,,
```

Comment:
I don't think that change is going to work - I expect that the LMS-DS Loader will fail with this date format. Looks like the file was opened and saved in Excel? Best to avoid that, since it will always mess with the date formatting.

This morning I found a VS Code extension that is very helpful for CSV files, just called "Edit csv"

Reply:
Thanks! I'll look into it and restore the date format